src/test/java/org/example/c03/controlFlow/NumberToWordsTest.java

```diff
@@ -0,0 +1,39 @@
+package org.example.c03.controlFlow;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class NumberToWordsTest {
+    @Test
+    public void given0ShouldReturnZero() {
+        int number = 0;
+        String myWords = "Zero";
```

Comment:
Give better name to myWords  variable. You can name it zero. Same for the rest of the tests in this class.

Reply:
done